js/addPagination.js

```diff
@@ -0,0 +1,72 @@
+import {
+  LOAD_MORE_BUTTON,
+  SHOW_LESS_BUTTON,
+  CLICK_EVENT,
+  PAGINATED_NO,
+} from "/js/constants.js";
+
+let currentItem = PAGINATED_NO;
+const container = document.getElementById("container");
+const itemsToShow = PAGINATED_NO;
+let visibleItems = itemsToShow;
+
+LOAD_MORE_BUTTON.addEventListener(CLICK_EVENT, function (event) {
+  event.preventDefault();
+  let boxes = [...document.querySelectorAll(".container .content .item")];
+  let numOfTaskPresent = Object.keys(boxes).length;
+  for (
+    var i = currentItem;
```

Comment:
Please do not use var as it often has side effect. Use block scoped `let` or `const`
```suggestion
    let i = currentItem;
```

Reply:
done!